app/views/shared/_post.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="user-posts">
+  <h2>Post #<%= post.id %></h2>
+  <p><%= truncate(post.text, length: 110, omission: " . . .") %></p>
+  <div class="like-and-comment">
+    <p>Comments: <%= post.comment_counter %>, Likes: <%= post.like_counter %></p> 
```

Comment:
**[OPTIONAL]**

- [ ] When I create a comment, like, or post then your update counter methods don't run :thinking: Kindly use `after_save` call back in the post, comment, and like model to run these methods after every insertion into the database :+1: